tests/mocha/block_test.js

```diff
@@ -2242,12 +2242,21 @@ suite('Blocks', function () {
         chai.assert.isTrue(blockB.disabled);
       });
       test('Disabled blocks from JSON should have proper disabled status', function () {
+        // Nested c-shaped blocks, inner block is disabled
         const blockJson = {
           'type': 'controls_if',
-          'enabled': false,
+          'inputs': {
+            'DO0': {
+              'block': {
+                'type': 'controls_if',
+                'enabled': false,
+              },
+            },
+          },
         };
         Blockly.serialization.blocks.append(blockJson, this.workspace);
-        const block = this.workspace.getTopBlocks(false)[0];
+        // Inner block
+        const block = this.workspace.getTopBlocks(false)[0].getChildren()[0];
```

Comment:
Nit:
```suggestion
        const innerBlock = this.workspace.getTopBlocks(false)[0].getChildren()[0];
```

Github doesn't let me rename it in the other places for some reason :/